fvm/evm/emulator/database/storable.go

```diff
@@ -0,0 +1,434 @@
+package database
+
```

Comment:
@fxamacker I wonder if we could expose these as part of standard or basic types as a sub-package in Atree, so we could import them instead of implementing them whenever we want to use them for basic use cases.

Reply:
@ramtinms 

Atree has some basic types implemented for testing only, so there is no problem if they use some constants that collide with Cadence types.  However, if atree expose those types, we want to make sure they don't overlap with Cadence types.  

Which basic types do you think we need atree to expose?